core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/kubernetes/reconfigure-rabbitmq-app.yml

```diff
@@ -3,3 +3,70 @@
   include_tasks: utils/patch-statefulset.yml
   vars:
     image_regexp: 'rabbitmq:.*'
+- name: Change rabbitmq stateful set to use {{ image_registry_address }}
+  block:
+    - name: upgrade-master | Get rabbitmq statefulset name
+      environment:
+        KUBECONFIG: /home/{{ admin_user.name }}/.kube/config
+      shell: |-
+        kubectl get statefulsets.apps --all-namespaces -o=jsonpath='{range .items[*]}{.metadata.name}{"\t"}{.spec.template.spec.containers[].image}{"\n"}{end}'|
+        grep -i rabbitmq |
+        awk '{print $1}'
+      changed_when: false
+      register: rabbit_mq_ss_name
+      args:
+        executable: /bin/bash
+    - name: upgrade-master | Get rabbitmq namespace
+      environment:
+        KUBECONFIG: /home/{{ admin_user.name }}/.kube/config
+      shell: |-
+        kubectl get statefulsets.apps --all-namespaces -o=jsonpath='{range .items[*]}{.metadata.namespace}{"\t"}{.spec.template.spec.containers[].image}{"\n"}{end}'|
+        grep -i rabbitmq |
+        awk '{print $1}'
```

Comment:
Same here

Reply:
pipe does not work with command

Reply:
yes, the same hare was about using only awk